Pod/Classes/PINRemoteImageManager.m

```diff
@@ -1070,12 +1070,13 @@ - (void)didReceiveData:(NSData *)data forTask:(NSURLSessionDataTask *)dataTask
         __weak typeof(self) weakSelf = self;
         [_concurrentOperationQueue pin_addOperationWithQueuePriority:PINRemoteImageManagerPriorityLow block:^{
             typeof(self) strongSelf = weakSelf;
-            PINImage *progressImage = [progressiveImage currentImageBlurred:shouldBlur maxProgressiveRenderSize:maxProgressiveRenderSize];
+            CGFloat imageProgress = 1.0;
```

Comment:
Should we call this imageQuality or something since the download progress (in terms of bytes received) could actually be greater at this point? Or does downloadProgress make sense to you in this context?